premarum-web-client/components/Landing/landingContact.tsx

```diff
@@ -140,7 +140,7 @@ export default function LandingContact() {
             TransitionComponent={isMobile? ({ children}) => children : Grow}
             sx={classes.snackbar}
         >
-            {successful?
```

Comment:
This was added to show the error one instead, since this was not yet implemented.